src/components/Room/Room.tsx

```diff
@@ -41,6 +45,19 @@ export function Room({
     roomId
   )
 
+  useEffect(() => {
+    const fetchSound = async () => {
+      audioBufferContainer.current = await fetch(
+        process.env.PUBLIC_URL + '/sounds/new-message.aac'
+      )
+        .then((response: Response) => response.arrayBuffer())
+        .then((arrayBuffer: ArrayBuffer) =>
```

Comment:
Would it be possible to use `await` instead of `then` here?

Reply:
okay :)